frontend/src/component/changeRequest/changeRequest.types.ts

```diff
@@ -44,7 +52,7 @@ export interface IChangeRequestComment {
     id: string;
 }
 
-export interface IChangeRequestBase {
+export interface IChangeRequestChangeBase {
```

Comment:
I'm thinking about creating a separate ticket for using generated types for those so that we have e2e type safety